src/plugins/filter/filter.js

```diff
@@ -37,11 +37,19 @@ var componentName = "wb-filter",
 		var elm = wb.init( event, componentName, selector ),
 			$elm, elmTagName, filterUI, prependUI,
 			settings, setDefault,
-			inptId, totalEntries;
+			inptId, totalEntries,
+			attr,
```

Comment:
You do not need to have `attr` if it is only for `maxWidth`. `maxWidth` will do the trick just fine.